pystruct/learners/n_slack_ssvm.py

```diff
@@ -12,13 +12,17 @@
 import cvxopt
 import cvxopt.solvers
 
-from sklearn.externals.joblib import Parallel, delayed
+from sklearn.externals.joblib import Parallel, delayed, cpu_count
 from sklearn.utils import gen_even_slices
 
 from .ssvm import BaseSSVM
 from ..utils import unwrap_pairwise, find_constraint
 
 
+def find_constraint_map(args):
+    return find_constraint(* args)
```

Comment:
you can create a new utils file, like ``utils.parallelization`` or something. yeah you could also pass a pool. I didn't really think through what would be the best interface.
Maybe we want a function that takes all the pool options and n_jobs and then decides which parallel thing to use. Or maybe have a utility function that takes a pool and put that in utils, and put a function that does the parallelization as a (private?) method into ``BaseSSVM``, like
```python
def _parallel_map(self, function, arguments):
    if self.pool is None:
        ...
```